db/structure.sql

```diff
@@ -101,6 +87,7 @@ CREATE TABLE advanced_searches (
 --
 
 CREATE SEQUENCE advanced_searches_id_seq
+    AS integer
```

Comment:
The data type is new in Postgres 10, and is optional. It is in my opinion preferable to not to have it in the dump, as it would break compatibility with Postgres 9.6 which is also ok for Catima.